index.php

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<?php
+require_once 'php-files/functions.php';
+
+$db = databaseConnect();
+$collection = getFish($db);
+?>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <link href="stylesheets/normalise.css" rel="stylesheet" type="text/css">
+    <link href="stylesheets/stylesheet.css" rel="stylesheet" type="text/css">
+    <title>Fish Finder</title>
+</head>
+<body>
+<div class="title-container">
+    <h1>Fish Finder</h1>
+</div>
+<div class="container">
+    <?php echo displayFish($collection); ?>
```

Comment:
Charlie's suggestion was that rather than running a function down here in the html body, load the output of this function into a variable in your php code at the top fo the document, and then echo the variable. It hink this is because if all the 'active' parts of your code are together then it's going to be easier to debug.